modules/repository-gcs/build.gradle

```diff
@@ -26,36 +26,36 @@ esplugin {
 }
 
 dependencies {
-  api 'com.google.cloud:google-cloud-storage:1.118.1'
-  api 'com.google.cloud:google-cloud-core:2.0.2'
-  api 'com.google.cloud:google-cloud-core-http:2.0.2'
-  runtimeOnly 'com.google.guava:guava:30.1.1-jre'
+  api 'com.google.cloud:google-cloud-storage:2.13.1'
+  api 'com.google.cloud:google-cloud-core:2.8.28'
+  api 'com.google.cloud:google-cloud-core-http:2.8.28'
+  runtimeOnly 'com.google.guava:guava:31.1-jre'
   runtimeOnly 'com.google.guava:failureaccess:1.0.1'
   api "commons-logging:commons-logging:${versions.commonslogging}"
   api "org.apache.logging.log4j:log4j-1.2-api:${versions.log4j}"
   api "commons-codec:commons-codec:${versions.commonscodec}"
-  api 'com.google.api:api-common:2.2.1'
-  api 'com.google.api:gax:2.0.0'
-  api 'org.threeten:threetenbp:1.5.1'
+  api 'com.google.api:api-common:2.3.1'
+  api 'com.google.api:gax:2.20.1'
+  api 'org.threeten:threetenbp:1.6.5'
   api 'com.google.protobuf:protobuf-java-util:3.17.3'
-  api 'com.google.protobuf:protobuf-java:3.21.1'
-  api 'com.google.code.gson:gson:2.8.9'
-  api 'com.google.api.grpc:proto-google-common-protos:2.3.2'
-  api 'com.google.api.grpc:proto-google-iam-v1:1.0.14'
-  api 'com.google.auth:google-auth-library-credentials:1.0.0'
-  api 'com.google.auth:google-auth-library-oauth2-http:1.0.0'
+  api 'com.google.protobuf:protobuf-java:3.21.7'
```

Comment:
I've also bumped `com.google.protobuf:protobuf-java` by a few patch versions, since it seems to have some CVEs.